app/event/index.js

```diff
@@ -1,9 +1,7 @@
-const sendSubmissionErrorEvent = require('./send-submission-error-event')
```

Comment:
Was this event not needed anymore?

Reply:
Correct, it did not add any value and wasn't used for anything.